### ISSUE TITLE
Feature request: Filter out categories, and filter out torrents with certain strings in the title

### ISSUE THREAD

**Issue by motbob** [ArjixGamer/Nyaa-Utility#8]
I wrote a crude script for this myself, and it's really a game-changer. Makes non-search browsing way cleaner.

**Comment by ArjixWasTaken** [comment]
Care to share that script?
Doing everything from scratch would take some time.

**Comment by motbob** [comment]
It's [here](https://gist.github.com/motbob/92af48bcf554427213ded5aa07a302c1), but I *really* don't think it's going to be helpful to you. I don't know anything about javascript at all.

**Comment by ArjixWasTaken** [comment]
Yeah it wasn't really helpful, but it helped me understand exactly what the feature you are requesting for is.

**Comment by ArjixWasTaken** [comment]
I'll add it tomorrow when I'm free, I have to go to sleep now.

**Comment by ArjixWasTaken** [comment]
wow, i completely forgot about this

**Comment by ArjixWasTaken** [comment]
ok, I added tags so far
Ill add categories on a later update
